core/tests/webdriverio_utils/DiagnosticTestPage.js

```diff
@@ -131,6 +191,32 @@ var DiagnosticTestPage = function () {
       expect(recommendedTopicSummaryTiles.length).toEqual(0);
     }
   };
+
+  var AutocompleteTopicDropdownEditor = async function (topicName) {
+    var containerLocator = '.e2e-test-classroom-category-dropdown';
+    var searchInputLocator = '.mat-select-search-input.mat-input-element';
```

Comment:
Please use a selector of the form `e2e-*`. If you need to, you can chain selectors as described in https://github.com/oppia/oppia/wiki/End-to-End-Tests#selecting-elements. If it's not possible to follow this convention here, please add a comment explaining why

Reply:
We can't do that in this case as it's external component. I have added a comment.

Reply:
Is the element also top-level in the DOM? If it has ancestor elements we can control, you can chain selectors

Reply:
Done I have chained it with e2e-test-exploration-new-category-add